lxd/network/openvswitch/ovs.go

```diff
@@ -373,3 +373,137 @@ func (o *OVS) OVNSouthboundDBRemoteAddress() (string, error) {
 
 	return addr, nil
 }
+
+// GetSTPPriority returns the STP priority of the OVS bridge.
+// Default STP priority is 32768 (0x8000). This value will be used if the
+// priority cannot be retrieved or if STP is disabled.
+func GetSTPPriority(ctx context.Context, bridgeName string) (string, error) {
+	output, err := shared.RunCommandContext(ctx, "ovs-vsctl", "get", "bridge", bridgeName, "other_config:stp-priority")
+	// get bridge does not differentiate in it's error codes between variable undefined and other errors, such as bridge not found
+	// the error output will need to be checked to make sure if stp-priority is not defined (defaults to 0x8000) or if a breaking error happened
+	if err != nil {
+		// default stp-priority is used if stp-priority is not defined
+		if strings.Contains(err.Error(), fmt.Sprintf("no key \"stp-priority\" in Bridge record \"%s\" column other_config", bridgeName)) {
+			return "8000", nil
+		}
+
+		// All other errors are considered errors
+		return "8000", err
+	}
+
+	// Convert to hex format
+	output = strings.TrimSpace(output)
+	// remove surrounding quotes
+	output = output[1 : len(output)-1]
+	stpPriority, err := strconv.Atoi(output)
+	if err != nil {
+		return "8000", err
```

Comment:
Please can you define a constant for this once rather than defining it multiple times.